# Generated by Django 4.0.3 on 2022-04-24 13:28

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion
import uuid


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='Favour',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(max_length=255, unique=True)),
                ('description', models.TextField()),
                ('price', models.DecimalField(decimal_places=2, max_digits=100)),
            ],
            options={
                'db_table': 'favours',
            },
        ),
        migrations.CreateModel(
            name='Job',
            fields=[
                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
                ('description', models.TextField()),
                ('started_at', models.DateTimeField()),
                ('ended_at', models.DateTimeField()),
                ('status', models.CharField(choices=[('Открыта', 'Opened'), ('В работе', 'In Progress'), ('Выполнена', 'Resolved'), ('Отложена', 'Postpone')], default='Открыта', max_length=100)),
                ('favour', models.ForeignKey(on_delete=django.db.models.deletion.RestrictedError, related_name='jobs', to='work_process.favour')),
                ('master', models.ForeignKey(on_delete=django.db.models.deletion.RestrictedError, related_name='jobs', to=settings.AUTH_USER_MODEL)),
            ],
            options={
                'db_table': 'jobs',
            },
        ),
    ]
